### ISSUE TITLE
Auto-Connect to Preferred PC on Launch

### ISSUE THREAD

**Issue by GeneBO98** [cgutman/moonlight-qt#1366]
**Is your feature request related to a problem? Please describe.**
Currently, there is no way that I can think of to get Moonlight to start a launch automatically with a Macro.  

**Describe the solution you'd like**
Incorporate an option to auto-connect to my "preferred" PC when Moonlight starts. This would make it more "console" like and make it easier to use in a living room setting.

**Describe alternatives you've considered**
I have attempted to use certain autohotkeys to launch Moonlight, then navigate to my PC, then launch the connection, but this is not reliable.

**Comment by jorys-paulin** [comment]
Hello! You can use the CLI to start streaming directly, see `moonlight --help`

**Comment by shodanx2** [comment]
Hello

For this I have created a shortcut

In target you put the CLI arguments

Here are mine

"C:\Program Files\Moonlight Game Streaming\Moonlight.exe" --display-mode windowed   --resolution 1280x720 stream 192.168.1.12 desktop

![image](https://github.com/user-attachments/assets/060fdcdb-00e1-4089-acfa-52456524368f)

I really wish I could disable mouse capture by default, because now I need to press CTRL+ALT+SHIFT+M every single time I restart it

**Comment by cgutman** [comment]
> I really wish I could disable mouse capture by default, because now I need to press CTRL+ALT+SHIFT+M every single time I restart it

You can with `--absolute-mouse`.

**Comment by malmaud** [comment]
IIUC we sitll don't have the ability to set a default host but not an application. Ideally `moonlight stream [host]` could skip directly to showing the apps for the given host.

**Comment by shodanx2** [comment]
@malmaud 
Actually you can specify the application, it is the very last argument
Here is my current commandline for a multiuser windows setup, works great

`Moonlight.exe --display-mode windowed -no-hdr --absolute-mouse  --resolution 1280x720 stream 192.168.1.12 desktop`

Now, I would love to have "RemoteApp" style support, but I could not yet get moonlight to ONLY show the app

Here is for example, trying to run "calc.exe"

![image](https://github.com/user-attachments/assets/4d0499c5-f8e0-42e8-9638-235664a18087)


It is still the full desktop, resizing the window, does not resize the application within, it only resizes the pixels of the stream

**Comment by malmaud** [comment]
Right, I want to be able to specify the host but NOT the application. When I tried leaving off the application, it didn't work.

**Comment by TuKo1982** [comment]
Is there an argument to shutdown the client after a disconnect?
